vmas/simulator/environment/environment.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         self.headless = None
         self.visible_display = None
         self.text_lines = None
+        self.visualize_semidims = kwargs.pop("visualize_semidims", True)
```

Comment:
This we can put in the Scneario class like the other rendering options

Reply:
Ok :)